### PR TITLE
[CPM-163] clear search bar when add course

### PR DESCRIPTION
**Pull request by HongShi10**
## Changes

- Clear searchbar when updating courses from populate API as well as editing a course
- Clear searchbar when deleting a course

## Checklist

- [] Have tests been written?
- [x] Is the UI consistent?

## Requests
- [ ] Can somebody check the styling/CSS?

Addresses [CPM-163]

Connects #y

[CPM-163]: https://momentum-agile.atlassian.net/browse/CPM-163

### CODE REVIEW

File: api/src/routes/course-route.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ router.delete("/:courseCode", async (req, res) => {
                 LOGGER.error("Course does not exist");
                 res.status(404).json({ msg: "Requested object not found" });
             } else {
-                LOGGER.info("DELETE request succeeded for /Course/${req.params.courseCode}");
+                LOGGER.info(`DELETE request succeeded for /Course/${req.params.courseCode}`);
```
Comment (daltonlim) [inline api/src/routes/course-route.js:202]: Do we have a test to check the correct message is printed?

Comment (daltonlim) [reply api/src/routes/course-route.js:202]: Might also need to change plan-route.js cause i mightve copied it from here

Comment (HongShi10) [reply api/src/routes/course-route.js:202]: uh how do u test logger statements?
I think it should be fine cause I remember I changed the other files maybe I just missed this one

Comment (daltonlim) [reply api/src/routes/course-route.js:202]: > uh how do u test logger statements?
> I think it should be fine cause I remember I changed the other files maybe I just missed this one

O nvm thought it was a response status. Don't think u can test loggers.
Yea plan-route.js is broke

File: client/src/pages/Courses/Courses.js [hunk 2]
```diff
@@ -123,13 +138,18 @@ const Courses = () => {
...
             {/* Right side of page */}
-            <Flex height="100%" width="50%" direction="column" >
+            <Flex height="100%" width="50%" direction="column">
                 <CourseView
                     course={selectedCourse}
                     isNew={isAddingCourse}
                     isEditing={isEditing || isAddingCourse}
-                    onEdit={() => setIsEditing(!isEditing)}
-                    onDelete={deleteCourse}
+                    onEdit={() => {
```
Comment (vee-christella) [inline client/src/pages/Courses/Courses.js:146]: Can we change this to a a single line without `{ }`

**Review by vee-christella — APPROVED** [review]
Nicee ✨ 💯

**Comment by Dinith1** [comment]
@pull-dog up

**Comment by ffMathy** [comment]
@pull-dog down

**Comment by ffMathy** [comment]
@pull-dog up

**Review by FocalChord — APPROVED** [review]
good work hong shi

**Pull request merged**